packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -139,6 +143,11 @@ const PersonalDetails = ({
         return is_svg ? localize('Last name*') : localize('Last name');
     };
 
+    const showAccountSettingsLinkOrText = () => {
+        if (is_virtual) return 'account settings';
+        return has_real_account ? '<0>account settings</0>' : 'account settings';
+    };
+
```

Comment:
can we simplify this as `const account_settings_text = has_real_account ? '<0>account_settings</0>' : 'account_settings';`?